pkg/config/config.go

```diff
@@ -255,6 +255,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("syslog_tls_verify", true)
 	config.BindEnvAndSetDefault("cmd_host", "localhost")
 	config.BindEnvAndSetDefault("cmd_port", 5001)
+	config.BindEnvAndSetDefault("api_config_port", 5004)
```

Comment:
The idea is for this port to serve much more data in the future. WDYT about `datadog_cross_process_port` or something like this (not a great name either, we might need to brainstorm it with the team) ?

We also need a dedicated host setting for this API.

Reply:
Considering the existing one is `ipc_port` (ie. "inter-process-communication-port") I'm not sure `cross_process_port` is clear enough to tell the difference 🤔 
Maybe `exposed_ipc_port` would be better for that ? (although "exposed" has a negative meaning security wise...)

Reply:
Following discussion with the team, going with `agent_ipc_host` and `agent_ipc_port`